pyproject.toml

```diff
@@ -1,8 +1,8 @@
 [tool.poetry]
 name = "yafti"
-version = "0.8.0"
+version = "0.9.0-beta1"
```

Comment:
This project currently uses release-please which will open new PRs based on semantic commits to increment versions. Generally we don't need to change the version here if we keep that workflow. Otherwise, you may want to disable that release-me job.

https://github.com/google-github-actions/release-please-action
https://github.com/ublue-os/yafti/pull/161

You can add a message in the body of a commit / PR that indicates the next release version needs to be something else. However, given the amount of work you're doing I'd keep the 0.x release pattern and do a 1.0 when you're happy. From there doing beta/alpha makes more sense but from a release pattern perspective 0.x semantic versions are by nature a beta already

Reply:
Agreed. Thanks for the docs!  

I added the beta just to make sure folks new it was not something ready for prime time.  What you mentioned above is/was my exact plan.